main.tex

```diff
@@ -331,21 +332,21 @@ \section*{Introduction}
 Depending on the application domain, different modeling approaches have their own documentation guidelines~\cite{Monks2018, Erdemir2012, Grimm2010}.
 In the case of systems biology, the \enquote{Minimum Information Requested in the Annotation of Biochemical Models (MIRIAM)}~\cite{LeNovere2005} and the \enquote{Minimum Information About a Simulation Experiment (MIASE)}~\cite{Waltemath2011} are two community standards used for documenting simulation models and corresponding simulation experiments.
 A recent perspective by Porubsky et al. (2020)~\cite{Porubsky2020} looks at all stages of a biochemical simulation study and at tools supporting their reproducibility.
-When looking at an entire simulation study and at the generation process of the included simulation model, these guidelines provide some indication about which information might be useful for documenting a complete simulation study as well as for establishing relationships between different simulation models.
+When looking at an entire simulation study and at the generation process of the included simulation model, these guidelines provide some indication about what information might be useful for documenting a complete simulation study as well as for establishing relationships between different simulation models.
 
 % Why do we have more than one model for a specific system and how can we relate these models?
 This is of particular interest when several simulation models for a system under consideration exist, offering different perspectives on the system, answering different questions, or reflecting the data and information available at the time of generation.
 Model repositories such as BioModels~\cite{BioModels2020, BioModels2018}, JWS Online~\cite{Snoep2004}, or the CellML Model Repository~\cite{Lloyd2008} provide different means to retrieve and use simulation models.
 % BioModels\footnote{\url{https://www.ebi.ac.uk/biomodels/}}, JWS Online\footnote{\url{https://jjj.biochem.sun.ac.za/}}, the CellML Model Repository\footnote{\url{https://models.cellml.org/}}
-For example, querying the BioModels database for biochemical and cellular simulation models that contain proteins such as \wnt{}, Janus kinase (Jak), or mitogen-activated protein kinase (MAPK), which are associated with corresponding signaling pathways, returns for \wnt{} a set of 22 simulation models, for Jak a set of 12 simulation models and for MAPK a set of 139 simulation models (as of January 2021).
+For example, querying the BioModels database for biochemical and cellular simulation models that contain proteins such as \wnt{}, Janus kinase (Jak), or mitogen-activated protein kinase (MAPK), which are associated with corresponding signaling pathways, returns 22 simulation models for \wnt{}, 12 simulation models for Jak and 139 simulation models for MAPK (as of January 2021).
```

Comment:
This sentence is fairly long and I think this modified to put the # of simulation models returned before the proteins (rather than after) simplifies it a bit